src/fountain.rs

```diff
@@ -601,7 +603,7 @@ impl Part {
 
 /// Calculates the quotient of `a` and `b`, rounding the results towards
 /// positive infinity.
-/// 
+///
```

Comment:
Can you squash this into the commit that introduced the function?

Reply:
Fixed